addons/base_import/models/base_import.py

```diff
@@ -803,14 +1094,16 @@ def _parse_import_data_recursive(self, model, prefix, data, import_fields, optio
                     for num, line in enumerate(data):
                         if re.match(config.get("import_image_regex", DEFAULT_IMAGE_REGEX), line[index]):
                             if not self.env.user._can_import_remote_urls():
-                                raise AccessError(_("You can not import images via URL, check with your administrator or support for the reason."))
+                                raise ImportValidationError(_("You can not import images via URL, check with your administrator or support for the reason."),
```

Comment:
Message should also be on new line if the other parameters are. Unless the indents are matched maybe (but that's a bit ugly).